docs/cicd/configure-an-sap-fiori-for-the-abap-platform-job-in-the-job-editor-4c26bfb.md

```diff
@@ -511,113 +511,83 @@ Depending on your configuration, the SAP Fiori for the ABAP platform pipeline ca
 
 ## \(Optional\) Configure the Additional Unit Test Stage
 
-Before running the **Additional Unit Tests** stage in your job, add the test configuration to your project.
-
-
+Before running the **Additional Unit Tests** stage in your job, add the Karma test configuration to your project.
 
 <a name="loio642bcb08b27c4d7ab5008aa277225189__prereq_fgb_ctn_blb"/>
 
 ## Prerequisites
 
 -   You’re an administrator of SAP Continuous Integration and Delivery.
 
--   In your repository, you have an SAPUI5/SAP Fiori project. See [Create an SAP Fiori Project](https://help.sap.com/viewer/9d1db9835307451daa8c930fbd9ab264/Cloud/en-US/46664de4d6944471b6c29a0681bfd0fc.html).
-
-
+-   In your repository, you have an SAPUI5/SAP Fiori project. See [Create an SAP Fiori Project](https://developers.sap.com/tutorials/appstudio-fioriapps-create.html).
```

Comment:
did you test this app with cicd and did it work out of the box with the below mentioned config?